platform/src/main/java/org/stellar/anchor/platform/component/custody/FireblocksBeans.java

```diff
@@ -69,4 +70,11 @@ CustodyPaymentService<TransactionDetails> custodyPaymentService(
       FireblocksApiClient fireblocksApiClient, FireblocksConfig fireblocksConfig) {
     return new FireblocksPaymentService(fireblocksApiClient, fireblocksConfig);
   }
+
+  @Bean
+  CustodyTransactionService getCustodyTransactionService(
```

Comment:
Rename to `custodyTransactionService` or `fireblocksCustodyTransactionService` to be constistent with other bean methods